### DISCUSSION TITLE
possible issue with utmstack.azurecr.io

### DISCUSSION THREAD

**Discussion by goetzk** [utmstack/UTMStack#794] [Q&A]
Hi,
Is this issue something specific to my site or utmastcks hosting?
```
Aug 26 03:40:23 utmstack dockerd[585]: time="2024-08-26T03:40:23.307849640Z" level=info msg="Attempting next endpoint for pull after error: Head \"https://utmstack.azurecr.io/v2/logstash/manifests/v10\": unauthorized: {\"errors\":[{\"code\":\"UNAUTHORIZED\",\"message\":\"authentication required, visit https://aka.ms/acr/authorization for more information.\"}]}"
Aug 26 03:40:23 utmstack dockerd[585]: time="2024-08-26T03:40:23.320122442Z" level=error msg="pulling image failed" error="Head \"https://utmstack.azurecr.io/v2/logstash/manifests/v10\": unauthorized: {\"errors\":[{\"code\":\"UNAUTHORIZED\",\"message\":\"authentication required, visit https://aka.ms/acr/authorization for more information.\"}]}" module=node/agent/taskmanager node.id=0fi6x3rea4t2t67bhfh77bvzj service.id=hucfu0ubhpbuucawrgxpti6b1 task.id=s8e7rfih082xbjawbdv70evip
```

My utmstack isn't coming up after installing and this is one of the recurring errors.

**Comment by goetzk** [comment]
I have belatedly found https://github.com/orgs/utmstack/discussions/368 which almost exactly matches my situation - I am running in an LXD (Qemu) VM which runs ubuntu 22.04.

It would be good to know why that authorisation fails in an LXD environment, so I will not close this just yet.

**Reply by c3s4rfred** [reply]
Hi, @goetzk let us check if there is a problem with azure's repository permissions, we'll coma back for you latter.

Best Regards

**Comment by c3s4rfred** [comment]
Hi, @goetzk we don't have support for LXC at current UTMStack's version, try installing it on a VM with full virtualization. We are migrating from azure, that will probably solve the criptography issues that causes to fail on LXD, but it's not available yet.

Best regards

**Comment by Kbayero** [comment]
As we don't have support for this, I will close the discussion.
